must-gather/node-gather/daemonset.yaml

```diff
@@ -11,6 +11,8 @@ spec:
       name: perf-node-gather-daemonset
   template:
     metadata:
+      annotations:
+        workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
```

Comment:
AFAIU must-gather is supposed to follow the same conventions as regular infrastructure pods, right?

Reply:
Correct.  Everything that's "OCP" is intended to be treated in the same workload / class of service.

Reply:
cool, then it seems we're good.